src/components/field-array-source/field-array-source.component.tsx

```diff
@@ -15,9 +15,26 @@ const options = [
 ];
 
 const handleClearKildebeskrivelse = form => {
+  form.values.kildebeskrivelse = null;
```

Comment:
What is the rationale for this? Doesn't the line underneath do exactly the same?

Reply:
Second line trigger a rerender.